agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -40,6 +40,7 @@
 #include <tlvf/wfa_map/tlvChannelSelectionResponse.h>
 #include <tlvf/wfa_map/tlvClientAssociationControlRequest.h>
 #include <tlvf/wfa_map/tlvClientAssociationEvent.h>
+#include <tlvf/wfa_map/tlvHigherLayerData.h>
```

Comment:
Please also remove this include from the `backhaul_manager.cpp` file if possible

Reply:
include removed from `backhaul_manager.cpp`.